### PR TITLE
Refactor networking api

### PR DESCRIPTION
**Pull request by NullRa**

### CODE REVIEW

File: Transportation_PLUS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -41,15 +43,17 @@
...
-		690A84F422B0A548009F4EC5 /* UbikeJson.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = UbikeJson.swift; sourceTree = "<group>"; };
 		690A84F522B0A548009F4EC5 /* MapManager.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = MapManager.swift; sourceTree = "<group>"; };
-		690A84F822B0A7B9009F4EC5 /* Station.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Station.swift; sourceTree = "<group>"; };
+		690A84F822B0A7B9009F4EC5 /* UbikeStation.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = UbikeStation.swift; sourceTree = "<group>"; };
+		6941055422E4ABBB00D90E22 /* CryptoAlgorithm.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = CryptoAlgorithm.swift; sourceTree = "<group>"; };
+		6941055E22E4ADA200D90E22 /* ptx.appID */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; name = ptx.appID; path = Common/ptx.appID; sourceTree = "<group>"; };
```
Comment (ender503) [inline Transportation_PLUS.xcodeproj/project.pbxproj]: Why the path of `ptx.appID` is changed?

Comment (NullRa) [reply Transportation_PLUS.xcodeproj/project.pbxproj]: It's OKay now !

File: Common/MainMapViewModel.swift [hunk 2]
```diff
@@ -122,7 +123,7 @@ class MainMapViewModel {
     }
 
     func showStationStatus(annotation: MKPointAnnotation?) -> String {
-        let ubkieDefault = UbikeJson()
+        let ubkieDefault = UbikeStationAPI()
```
Comment (ender503) [inline Common/MainMapViewModel.swift]: It would be better to rename the variable to `ubikeStationAPI` or `ubikeAPI`. 🤔

**Comment by ender503** [comment]
In the commit message, api should be upper-case `API`.

**Comment by ender503** [comment]
Rebasing is required. 😏

File: Common/MainMapViewModel.swift (hunk 2 above)
Comment (NullRa) [reply Common/MainMapViewModel.swift]: got it!

**Pull request merged**